Sources/GDBRemote/Structures.cpp

```diff
@@ -165,7 +166,35 @@ std::string ProcessThreadId::encode(CompatibilityMode mode) const {
   return ss.str();
 }
 
-void StopInfo::reasonToString(std::string &key, std::string &val) const {
+void StopInfo::getWatchpointInfo(std::string &key, std::string &val,
+                                 CompatibilityMode mode, bool encodeHex) const {
```

Comment:
That's not actually true (this is one of those conversations we had way back when while I was writing this). The problem is that in a qThreads vs jThreads packet (both lldb mode), these need to be hex vs non-hex encoded.